Content.Shared/Chat/SharedChatSystem.cs

```diff
@@ -15,6 +15,7 @@ public abstract class SharedChatSystem : EntitySystem
     public const char LOOCPrefix = '(';
     public const char OOCPrefix = '[';
     public const char EmotesPrefix = '@';
+    public const char EmotesAltPrefix = '*';
```

Comment:
As said before, to future proof this, is there a big problem with making these char arrays?

Reply:
technically no but it needs a huge chunk of rewriting chat code, which is outside the scope of this PR (i'm not going to refactor chat for this)

```
    // the following code is problematic in SharedChatSystem
    public static readonly char[] RadioCommonPrefix = ";".ToCharArray();  // EncryptionKeySystem.cs:223 makes `key` nullable
    public static readonly char[] RadioChannelPrefix = ":".ToCharArray(); // same location, same nulling
```